github/repos_contents.go

```diff
@@ -72,7 +73,8 @@ func (r *RepositoryContent) Decode() ([]byte, error) {
 	return o, nil
 }
 
-// GetReadme gets the Readme file for the repository.
+// GetReadme gets the Readme file for the repository, which is stored as base64.
```

Comment:
These methods clearly return a `RepositoryContent` struct, so wouldn't it be reasonable to expect a user to look at that type to then know what to do with it?

Nowhere else in go-github (that I can recall), does a method document how to use the returned type.  Instead, the type itself documents what its fields are, special cases about when they may or may not be present, etc.  I think the one exception may be where we mention that the caller is responsible for closing an `io.ReadCloser` similar to how the `net/http` documents that, but I may be misremembering that.